ompi/mca/common/ompio/common_ompio_buffer.c

```diff
@@ -144,7 +148,7 @@ void *mca_common_ompio_alloc_buf ( ompio_file_t *fh, size_t bufsize )
     if ( !mca_common_ompio_buffer_init ){
         mca_common_ompio_buffer_alloc_init ();
     }
-    
```

Comment:
Random whitespace changes throughout this commit, can you split whitespace changes to a separate commit?

Reply:
removed commit